microsoft-edge/webview2/get-started/wpf.md

```diff
@@ -343,6 +343,9 @@ To demonstrate how to use the events, register a handler for `NavigationStarting
 ## Step 8 - Scripting
 
 You can use host apps to inject JavaScript code into WebView2 controls at runtime.  You can task WebView2 to run arbitrary JavaScript or add initialization scripts.  The injected JavaScript applies to all new top-level documents and any child frames until the JavaScript is removed.
+<!-- TODO: are Core and Framework both supported?
+Core and Framework are both supported.
+-->
```

Comment:
@nishitha-burman - are Core and Framework both supported for scripting here?
Rendered section:
https://review.docs.microsoft.com/en-us/microsoft-edge/webview2/get-started/wpf?branch=pr-en-us-2135#step-8---scripting